Control/Monad/Error/Class.hs

```diff
@@ -190,3 +194,21 @@ instance (Monoid w, MonadError e m) => MonadError e (LazyWriter.WriterT w m) whe
 instance (Monoid w, MonadError e m) => MonadError e (StrictWriter.WriterT w m) where
     throwError = lift . throwError
     catchError = StrictWriter.liftCatch catchError
+
+-- | MonadError analog to the 'try' function.
```

Comment:
I don't think `'try'` will be hyperlinked in the rendered Haddocks, since `Control.Exception` doesn't appear to be imported. I think the usual workaround for this problem is to `import qualified Control.Exception` and use `'Control.Exception.try'` in the Haddocks.

Reply:
resolved